backend/typescript/services/implementations/__tests__/campService.test.ts

```diff
@@ -27,8 +30,16 @@ const testCamps: CreateCampDTO[] = [
     fee: 25,
     campSessions: [],
     formQuestions: [],
+    campCoordinators: ["61fb3d34272ea0002ad6a24d"],
```

Comment:
this entire time i thought mongodb did foreign key checks. did not know we can just stick in any object id to all of our reference fields. i'm sad